.pipelines/v2/templates/steps-ensure-dotnet-version.yml

```diff
@@ -21,7 +23,7 @@ steps:
       exit 1
     }
     $NEW_DOTNET_ROOT = "$(Agent.ToolsDirectory)\dotnet"
-    & ./dotnet-install.ps1 -Channel "${{parameters.version}}" -InstallDir $NEW_DOTNET_ROOT
+    & ./dotnet-install.ps1 -Channel "${{parameters.version}}" -Architecture ${{parameters.platform}} -InstallDir $NEW_DOTNET_ROOT
```

Comment:
@crutkas As far as I can tell, this was previously installing .NET x64 on the ARM job. Is it expected that ARM64 isn't installed?

Reply:
@dhowett, i'm betting we just didn't know this was a needed param?  @Youssef1313  we rewrote a lot of the pipeline a few months ago

Reply:
I'm not actually sure whether this is needed or not. I'm still experimenting and trying to understand.

Reply:
I think the issue is more about having an arm64 agent. Then dotnet-install will auto-detect and install the right thing.